src/main/kotlin/com/openlattice/chronicle/controllers/StudyController.kt

```diff
@@ -428,6 +429,16 @@ class StudyController @Inject constructor(
         return OK()
     }
 
+    @GetMapping(
+        path =[STUDY_ID_PATH + FEATURES_PATH],
+        produces = [MediaType.APPLICATION_JSON_VALUE]
+    )
+    override fun getStudyFeatures(@PathVariable(STUDY_ID) studyId: UUID): Map<StudyFeature, Map<String, Any>> {
```

Comment:
Should we just return Map<String, Any> or keep it consistent with a bulk endpoint? Cc @UnsungHero97